homeassistant/components/tuya/vacuum.py

```diff
@@ -184,3 +186,7 @@ def locate(self, **kwargs: Any) -> None:
     def set_fan_speed(self, fan_speed: str, **kwargs: Any) -> None:
         """Set fan speed."""
         self._send_command([{"code": DPCode.SUCTION, "value": fan_speed}])
+
+    def send_command(self, command, params=None, **kwargs):
```

Comment:
Please add typing, as requested in my last review.

Reply:
Sorry. It's already fixed.

Reply:
Unresolved this comment.

Please fully type this method, that includes all parameters of this method.